Documentation/rkt-build-rkt.md

```diff
@@ -0,0 +1,111 @@
+# coreos.com/rkt/builder
+
+This container contains all build-time dependencies in order to build rkt. It
+currently can be built in one of two flavors: _Debian Sid_ or _Fedora 22_.
+
+All commands assume you are running them in your local git checkout of rkt.
+
+## Building coreos.com/rkt/builder using acbuild
+
+Requirements:
+- Go or `docker2aci`
+- rkt
+
+The file `scripts/acbuild-rkt-builder.sh` contains a simple bash script which
+will download a base docker image which is then converted into an ACI. Once the
+ACI has been downloaded, the manifest is patched. Next acbuild is run which
+creates an ACI that has all the required build dependencies for compiling `rkt`.
+
+If you want to change the base OS you can set using one of the following sets of
```

Comment:
I wanted to be clear that you can mix and match those values (ie: `fedora sid` isn't valid). I can change this to be more clear.